scss/bootstrap/overrides/_variables.scss

```diff
@@ -370,3 +370,5 @@ $blockquote-small-color:         $crm-copy !default;
 $blockquote-border-color:        $brand-primary !default;
 
 $crm-ui-select-top-padding:      4px !default;
+
+$hr-border: $crm-grayblue-dark !default;
```

Comment:
unnecessary, just use the variable directly

Reply:
@AkA84 This is for overriding bootstrap variable, `$hr-border` this is bootstrap scaffolding variable for `<hr>` tag.